apps/coscrad-frontend-e2e/src/e2e/note-creation/create-note.e2e.cy.ts

```diff
@@ -0,0 +1,68 @@
+import { ResourceType } from '@coscrad/api-interfaces';
+
+describe(`note creation`, () => {
+    const resourceType = ResourceType.term;
```

Comment:
Is it worth looping over all resource types?

Reply:
Good question, are we vulnerable if not?

Reply:
It's an interesting question. A purist approach would be to create such a test for every resource type. This assumes nothing about the implementation. But the time cost of running those tests is huge.

Knowing that the note command form is always the same, we could do a sanity check that the form is rendered on the detail view for every resource and be fairly confident. 

This is related to the question of whether this should be a standalone `create note` test suite, or whether we should have this as a test case for every resource detail admin view.

Reply:
I've sorted this out. We will keep the one super-detailed test for create note for term. We will add a sanity-check simplified e2e test for all other resource types.